kubeedge-counter-demo/README.md

```diff
@@ -49,7 +49,8 @@ The KubeEdge Web App runs in a VM on cloud.
 $ cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/web-controller-app
 $ make
 $ make docker
-$ cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/templates
+$ cd $GOPATH/src/github.com/kubeedge/examples/kubeedge-counter-demo/crds
+$ kubectl taint nodes --all node-role.kubernetes.io/control-plane-
```

Comment:
this is required to delete, since the following will try to deploy the container to the master control-plane.

https://github.com/kubeedge/examples/blob/9ce0a1c82deff9f2587ff4a01fa083a6c64227af/kubeedge-counter-demo/crds/kubeedge-web-controller-app.yaml#L18-L19

see also, https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#control-plane-node-isolation

instead, the following error will happen.

```bash
# kubectl get pods -o wide
NAME                                   READY   STATUS    RESTARTS   AGE     IP       NODE     NOMINATED NODE   READINESS GATES
kubeedge-counter-app-7d4f89d55-hkz5h   0/1     Pending   0          3m39s   <none>   <none>   <none>           <none>

# kubectl describe pods kubeedge-counter-app-7d4f89d55-hkz5h
...<snip>
Events:
  Type     Reason            Age    From               Message
  ----     ------            ----   ----               -------
  Warning  FailedScheduling  2m46s  default-scheduler  0/2 nodes are available: 1 node(s) had untolerated taint {node-role.kubernetes.io/control-plane: }, 2 node(s) didn't match Pod's node affinity/selector. preemption: 0/2 nodes are available: 2 Preemption is not helpful for scheduling.
```

Reply:
If we want to deploy `kubeedge-web-controller-app` to `cloud` side, so  how about add another toleration instead of removing taint from master node?

Reply:
+1, agree. that way, it does not taint the master node or configuration that affect the cluster system.

Reply:
So let's update the [kubeedge-web-controller-app.yaml](https://github.com/kubeedge/examples/blob/9ce0a1c82deff9f2587ff4a01fa083a6c64227af/kubeedge-counter-demo/crds/kubeedge-web-controller-app.yaml#L18-L19) in this PR?

Reply:
@fisherxu https://github.com/kubeedge/examples/pull/122/commits/688c816e3b1f9f1135170c170feb2247f5877fdd can address your suggestion.

Reply:
Looks good to me :)